list.go

```diff
@@ -336,14 +340,21 @@ func (api *API) DeleteList(ctx context.Context, rc *ResourceContainer, listID st
 // API reference: https://api.cloudflare.com/#rules-lists-list-list-items
 func (api *API) ListListItems(ctx context.Context, rc *ResourceContainer, params ListListItemsParams) ([]ListItem, error) {
 	var list []ListItem
-	var cursor string
-	var cursorQuery string
+
+	var query = url.Values{}
```

Comment:
within the library, we can use a combination of `buildURI` and struct tags to simplify the handling and building of URLs. seeing how there is a release planned for tomorrow, i pushed up these changes on your behalf in 1cad030 to get this ready for merging. do let me know if you have any questions!